Makefile

```diff
@@ -265,8 +265,12 @@ test-e2e:
 	@$(MAKE) test-e2e-aws
 	@$(MAKE) test-e2e-gcp
 test-e2e-aws:
+	@$(MAKE) images-all-aws
+	@$(MAKE) operator-update-aws
```

Comment:
nit: I think this can be `operator-restart-aws` since the operator image will be built and pushed in the previous line (also for GCP)

Reply:
That makes sense!